lib/plugins/aws/deploy-function.js

```diff
@@ -313,6 +337,26 @@ class AwsDeployFunction {
           }
         });
       }
+      if (hasServerlessConsoleSDK || hasServerlessConsoleExternal) {
+        const { environmentKeys } = await apiRequest('/api/integrations/aws/layer/config', {
+          urlName: 'integrationsBackend',
+          noAuth: true,
+        });
```

Comment:
Yeah I was a bit torn here. I don't think our environment variable schema has changed much since we started building our layers so hard coding should probably be alright.

I just wasn't sure if we were planning on adding any variables in the future especially with features like sampling coming soon.

I guess I will hard code them for now 👍